core/src/toga/style/pack.py

```diff
@@ -764,9 +764,10 @@ def __css__(self):
         # direction
         css.append(f"flex-direction: {self.direction.lower()};")
         # flex
-        if (self.width == NONE and self.direction == ROW) or (
-            self.height == NONE and self.direction == COLUMN
-        ):
+        if (
+            (self.width == NONE and self.direction == ROW)
+            or (self.height == NONE and self.direction == COLUMN)
+        ) and getattr(self, "_use_default_width", True):
```

Comment:
For posterity: this approach was developed pairing with @ahter. 

The underlying problem is that if the ActivityIndicator has no Toga style, it falls back to rendering as `flex: 0 0 0px`, which evaluates as 0 width in CSS. If you apply a fixed width of 20px, the animation for the spin and the gray background become misaligned. We need to work out how to:
1. Apply a style override to the ActivityIndicator so that it doesn't collapse to 0 size, or
2. Work out how to inject a widget-specific modification to Pack's `__css__` implementation to disable the default 

It feels to me like both are needed. (1) is needed because if a user applies a "width=100" pack style, we don't want the widget to render weird; (2) is needed to make sure that the default style results in a visible widget.